csrc/executor.h

```diff
@@ -164,11 +170,26 @@ class FusionExecutor : public NonCopyable {
     post_lowering_hooks_.push_back(std::move(hook));
   }
 
+  // Function to query whether compilation was attempted for a `FusionExecutor`
+  bool isCompiled() const {
```

Comment:
looks like we changed the plan of renaming here.

Reply:
I'm not nitpicking on this BTW.

Reply:
Yes, I decided to keep `isCompiled` the same at every place. At `FusionExecutorCache / FusionKernelRuntime` level, we should not need to distinguish between nvFuser vs ATen evaluation generally, so trying to limit that detail within `FusionExecutor`